src/main/java/org/isf/exa/gui/ExamEdit.java

```diff
@@ -249,49 +241,47 @@ private JButton getOkButton() {
 				okButton = new JButton();
```

Comment:
line 239 and line 240 are the same; one of them can be removed

Reply:
again not my part of code and I think it makes no sense to fix everything on gui side. It would be easier to write it from scratch :)